docs/markdown/how-to.md

```diff
@@ -0,0 +1,131 @@
+---
+title: How to work with us
+description: Guidelines on how to engage with the Gestalt team, when to work with us, and how to contribute.
+---
+
+## What goes into Gestalt?
+
+The goal of Gestalt is to create a shared library of design best practices, React components, and documentation. Therefore, the best contenders for addition to Gestalt are designs or components that benefit multiple teams and are used regularly throughout our products.
+
+## Requesting new components or changes
+
+**Please note:** The process below applies to Pinterest employees.
+We ask that any ideas that change the functionality of an existing Gestalt component or introduce a net-new component go through the following process:
+
+1. **Present work early in our [partnership meetings](#Meetings-and-events)** \
+   What we build into Gestalt comes from the teams across Pinterest, so we’d love to pair with you early and often! Seeing early mocks, wireframes, or even product roadmaps and learning where your project may be going helps inform our team OKRs. The earlier we see the work, the better we're able to plan and the higher the likelihood we'll be able to help. Please [sign up](https://pinch.pinadmin.com/gestaltSignUp) for an Office Hours slot with your discussion topic.
```

Comment:
Can we get links to open in a new tab? In a way we can control on a callsite basis?

Reply:
We can, I'd probably make a custom `Link` component